websites/R/Reddit/presence.ts

```diff
@@ -83,13 +83,18 @@ presence.on("UpdateData", async () => {
       presenceData.smallImageKey = "live";
       presenceData.smallImageText = (await strings).live;
     } else {
-      subReddit = document.querySelector(
+      delete presenceData.smallImageKey;
+      const sub = document.querySelector(
         "span._1GieMuLljOrqnVpRAwz7VP"
-      ).textContent;
+      );
+      if (sub === null) {
+       sub = document.querySelector("#SHORTCUT_FOCUSABLE_DIV > div:nth-child(4) > div > div > div > div._3ozFtOe6WpJEMUtxDOIvtU > div.q4a8asWOWdfdniAbgNhMh > div > div.QscnL9OySMkHhGudEvEya > div > div._3TG57N4WQtubLLo8SbAXVF > h2");
+        presenceData.details = (await strings).browsing;
+        presenceData.state = !sub ? "Home" : sub.textContent;
+      } else {
       presenceData.details = (await strings).browsing;
-      presenceData.state = subReddit;
-      delete presenceData.smallImageKey;
-    }
+      presenceData.state = sub.textContent;
+    } }
```

Comment:
is this double close correct?
cant really see it because you didnt prettify it..

Reply:
yes